src/main/java/at/hannibal2/skyhanni/config/features/garden/GardenConfig.java

```diff
@@ -211,15 +216,6 @@ public class GardenConfig {
     @FeatureToggle
     public boolean jacobContestSummary = true;
 
-    @Expose
-    @ConfigOption(
-        name = "Personal Best Increase FF",
-        desc = "Show in chat how much more FF you get from farming contest personal best bonus after beating the previous record."
-    )
-    @ConfigEditorBoolean
-    @FeatureToggle
-    public boolean contestPersonalBestIncreaseFF = true;
```

Comment:
because you are moving this option, you need to add a config fix so that if people turned it off before it wont turn back on, search the codebase for examples

Reply:
fixed

Reply:
should actually be fixed now